test-two/src/index.js

```diff
@@ -1,19 +1,44 @@
 import { log } from "./helpers/logger";
 import { getUsersForOrganisation } from "./services/github";
 
+
+
 const baseUrl = "https://api.github.com/";
 const organisationId = "google";
-const gitHubApiAuthToken = ""; // add your GitHub API OAuth key here to increase request limit
+const gitHubApiAuthToken = process.env.ACCESSS_TOKEN; // add your GitHub API OAuth key here to increase request limit
+
+const MAX_PER_PAGE = 100; // max users per page is 100 ; default is 30
+var page = 1; // start from first page
+const allUsers =[]; // initialise the array to store all users in it
 
 export const go = async () => {
   try {
-    const users = await getUsersForOrganisation(
+    var users = await getUsersForOrganisation(
```

Comment:
there a reason you're using `var` instead of es6 `const` or `let`?

Reply:
I use let and var interchangeably as there is no significant difference between them but in this case, I sticked to var, the reason I did not use const as I am fetching the users inside the while loop thus users value changes every time until we fetch all users. I think I could have just declared the value inside the while loop?